kits/notion-assistant/constitutions/default.md

```
@@ -0,0 +1,18 @@
+# Notion Assistant Constitution
+
+## Core Principles
+
+### 1. Purpose and Persona
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)